src/i18n/languages.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   'en-US': 'English',
-  // 'de-DE': 'Deutsch',
+  'de-DE': 'Deutsch',
```

Comment:
Assuming that this will be the only or main german translation `de` as key is sufficient. 
I will update this once it's merged.